### DISCUSSION TITLE
display net speed on LXQT panel

### DISCUSSION THREAD

**Discussion by thepriyanshujangid** [lxqt/lxqt#2600] [Q&A] [answered]
Can we have  applet  to check our  outgoing and incomming internet speed  same  as we have in  xubuntu panel.
as  indicator-multiload does not work th Lubuntu 18.04+  versions.

**Comment by stefonarch** [comment]
Add "sysstat" plugin and configure it to "network" and your connection (wireless/eth)?

**Reply by thepriyanshujangid** [reply]
I use both.
In my desktop i use eth and on my laptop i use wifi.
It shows graph, it does not show number i mean what Mb/Kb speed we are getting.


Whereas we have  an option in xubuntu to setup  this 
I dont remember the exact name but it is something like  Value in bar  something like that.

**Reply by Bluey26** [reply]
I use a conky dock in the desktop with those values, plus other things(cpu usage,ram usage, I/O to disk,etc)

**Reply by stefonarch** [reply, marked as answer]
You could use some command output in "custom command" plugin like
`ifstat | grep enp2s0f3u2u4 | awk '{print "Down: "$2" KB/s Up: "$3" KB/s"}'`

![immagine](https://github.com/user-attachments/assets/1694ed96-b7d8-4d52-96fd-5e42a26c4a66)
You need to replace enp2s0f3u2u4 with your interface. Maybe there are better solutions.

**Reply by thepriyanshujangid** [reply]
![image](https://github.com/user-attachments/assets/eb6922cf-1cd4-4cb0-a01e-5484742c85f7)

thank u

**Reply by thepriyanshujangid** [reply]
![image](https://github.com/user-attachments/assets/a0a700dc-27e9-4c97-a164-b74a206c7413)

btw,  this one  looking good.  xD 
i thought   just to share this one.

**Reply by thepriyanshujangid** [reply]
ifstat -i enx16508b731974 1 1 | awk 'NR==3 {print "⬇ "$1" KB/s    ⬆ "$2" KB/s"}'

**Reply by stefonarch** [reply]
`ifstat: invalid option -- 'i'`on arch
Btw I changed this to Q&A so an answer can be marked.
